static/js/babelhook.js

```diff
@@ -7,7 +7,7 @@ require('jsdom-global')(undefined, {
 });
 
 // We need to explicitly change the URL when window.location is used
-const changeURL = require('jsdom').changeURL;
+const changeURL = require('jsdom/lib/old-api').changeURL;
```

Comment:
I'm fine with this as is but is there an easy way to use the new jsdom API, or is that too much work for this PR?